style.css

```diff
@@ -278,3 +287,113 @@ nav a:hover {
 hr {
   color: #dfe1e6;
 }
+
+form {
+  margin-top: 40px;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: url("./assets/images/contactform-shapes.png");
+  border-top-left-radius: 60px;
+  padding: 74px 5%;
+  gap: 12px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 222px 759px;
+  background-position-x: right;
+  background-position-y: 23px;
+  width: auto;
+  height: 100%;
+  background-color: #6070ff;
+}
+
+.contact h2 {
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  font-style: bold;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-top: 44px;
+  vertical-align: center;
+}
+
+.contact p {
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: regular;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  vertical-align: top;
+  color: #ebebff;
+}
+
+.contact input,
+textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 15px 10px 15px 10px;
+  margin: 12px 0 12px 0;
+  width: 300px;
+  text-align: left;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+}
+
+#submit {
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: 1px;
+}
```

Comment:
Please kindly implement the button interactions according to the project requirements to make your design similar to [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🤗

![image](https://user-images.githubusercontent.com/78251927/235780770-d8ee7dc6-db75-4e42-8669-b1112c796b9a.png)

![image](https://user-images.githubusercontent.com/78251927/235780897-58eb3624-a4c4-4df8-9a1a-be0bbdd3194d.png)

Reply:
@abdulhamiid I implemented the button interactions if you can see on the next line having a .fcontact .btn:hover . Please kindly check it and correct me.